src/pages/[platform]/build-a-backend/add-aws-services/analytics/personalize-recommendations/index.mdx

```diff
@@ -0,0 +1,137 @@
+
+To record event data, you need the following:
+
+* [A dataset group](https://docs.aws.amazon.com/personalize/latest/dg/data-prep-ds-group.html)
+* [An event tracker](https://docs.aws.amazon.com/personalize/latest/dg/event-get-tracker.html).
+
+For more information, see [Record Events](https://docs.aws.amazon.com/personalize/latest/dg/recording-events.html).
+
+### Installation and Configuration
+
+After creating the Amazon Personalize dataset group, you need to add the `personalize:PutEvents` permission to your Amplify IAM user roles.
+
+An example IAM policy:
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [{
+    "Effect": "Allow",
+    "Action": "personalize:PutEvents",
+    "Resource": "*"
+  }]
+}
+```
+
+You need the tracking ID of your event tracker. For more information, see [Get a Tracking ID](https://docs.aws.amazon.com/personalize/latest/dg/action-interaction-tracker-id.html).
+
+Configure Amazon Personalize:
+
+```javascript
+import { Amplify } from 'aws-amplify';
+
+Amplify.configure({
+  ...Amplify.getConfig(),
+  Analytics: {
+    Personalize: {
+      // REQUIRED - The trackingId to track the events
+      trackingId: '<TRACKING_ID>',
+
+      // REQUIRED -  Amazon Personalize service region
+      region: 'us-east-1',
+
+      // OPTIONAL - The number of events to be deleted from the buffer when flushed.
+      flushSize: 10,
+
+      // OPTIONAL - The interval in milliseconds to perform a buffer check and flush if necessary.
+      flushInterval: 5000 // 5s
+    }
+  }
+});
+```
+### Working with the API
+
+You can use the `Identify` event type to track a user identity. This lets you connect a user to their actions and record traits about them. To identify a user, specify a unique identifier for the userId property.
+ Consider the following user interactions when choosing when and how often to call record with the  Identify eventType:
+
+* After a user registers.
+* After a user logs in.
+* When a user updates their information (For example, changing or adding a new address).
+* Upon loading any pages that are accessible by a logged-in user (optional).
+
+```javascript
+import { record } from 'aws-amplify/analytics/personalize';
+
+record({
+  eventType: "Identify",
+  properties: {
+     userId: "<USER_ID>"
+  }
+});
+```
+You can send events to Amazon personalize by calling the `record` operation. If you already use `Identify` to track end-user data, you can skip the userId, the SDK will fetch the userId based on current browser session.
+For information about the properties field, see [Put Events](https://docs.aws.amazon.com/personalize/latest/dg/API_UBS_PutEvents.html).
+
+```javascript
+import { record } from 'aws-amplify/analytics/personalize';
+
+record({
+  eventType: "<EVENT_TYPE>",
+  userId: "<USER_ID>", // optional
+  properties: {
+    itemId: "<ITEM_ID>",
+    eventValue: "<EVENT_VALUE>"
```

Comment:
nit placeholder text should be lowercase kebab-case

Reply:
personalize schema are in upper case

```
{
  "type": "record",
  "name": "Interactions",
  "namespace": "com.amazonaws.personalize.schema",
  "fields": [
    {
      "name": "USER_ID",
      "type": "string"
    },
    {
      "name": "ITEM_ID",
      "type": "string"
    },
    {
      "name": "TIMESTAMP",
      "type": "long"
    },
    {
      "name": "EVENT_TYPE",
      "type": "string"
    }
  ],
  "version": "1.0"
}
```